src/Builder/FormContractor.php

```diff
@@ -111,7 +116,7 @@ public function getDefaultOptions($type, FieldDescriptionInterface $fieldDescrip
                 $options['model_manager'] = $fieldDescription->getAdmin()->getModelManager();
 
                 break;
-            case 'Sonata\AdminBundle\Form\Type\Modeltype':
+            case 'Sonata\AdminBundle\Form\Type\ModelType':
```

Comment:
@dbu this was the reason, why we had to set `ModelManager` on our `ModelType` forms :-)